lang/nl/manage-products/products.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+return [
+    "title_label" => "Producten",
+    "groups_heading" => "Groepen",
+    "actions_heading" => "Actie",
+    "action_edit_label" => "Wijzig",
+    "action_delete_label" => "Verwijder",
+    "action_edit_class" => "edit-product-button",
```

Comment:
?

Reply:
Alles met "class" erbij is voor de test keys.
@adjagers Dit lijkt mij eigenlijk niet nodig of misschien dit niet in de Nederlandse language files zetten, het zijn testen namelijk